frontend/templates/views/partials/component-headline.j2

```diff
@@ -13,15 +13,26 @@
 {% endif %}
 </h1>
 
-{% if doc.experimental or doc.deprecated %}
+{% if doc.experimental or doc.deprecated or doc.bento %}
```

Comment:
Would it make sense to add bento to the `formats` field?

Reply:
Could make sense. Is there a specific plan how bento component docs will be structured? Or do they have API parity with the classic components and therefore don't need their own docs but really just a marker?

Wasn't comfortable making assumptions about this and therefore simply added the flag 😁

Reply:
Bento and AMP component docs are going to be the same (Bento and AMP version of a component is the same JS script). So a flag is fine.